src/hooks/reactQuery/useFetchDailyRevenue.ts

```diff
@@ -0,0 +1,12 @@
+import { useQuery } from '@tanstack/react-query';
+import dayjs from 'dayjs';
+import { DashboardService, FilterDateParams } from 'stores';
+
+export const useFetchDailyRevenue = (params: FilterDateParams) => {
```

Comment:
please return a default value 
export const useFetchDailyRevenue = (params: FilterDateParams) => {
  return useQuery({
    queryKey: ['dashboard-dailyRevenue', dayjs(params.start).format('L'), dayjs(params.end).format('L')],
    queryFn: async () => {
      try {
        const data = await DashboardService.instance.fetchDailyRevenue({ start: params.start, end: params.end });
        return data;
      } catch (error) {
        // Handle the error and return a default value
        console.error('Failed to fetch daily revenue:', error);
        return { revenue: 0 }; // Return a default value or structure
      }
    },
    select: (data) => {
      // If data is undefined or null, return a default value
      return data || { revenue: 0 };
    },
  });
};